raaghu-elements/src/rds-progress-bar/rds-progress-bar.tsx

```diff
@@ -12,6 +12,8 @@ export interface RdsProgressBarProps {
     displayLabel?: boolean;
     displayPercentage?: boolean;
     width?: string;
+    State?: string;
+    state?: string;
```

Comment:
remove unused props

Reply:
@hemalatagaikwad these props are used to change the state for default, inporgress, completed and error we cant remove that 

![image](https://github.com/user-attachments/assets/2cb2dc90-ab60-49b1-b933-961212e31331)